src/project/views/NewUserView.java

```diff
@@ -46,18 +50,24 @@ public static void newUserView(Stage stage){
 		provinceBox.add("QC");
 		provinceBox.add("SK");
 		provinceBox.add("YT");
+		String province = provinceCombo.getValue().toString();
 		Label postCodeLabel = new Label("Postal Code:");
 		postCodeField = new TextField();
+		String postalCode = postCodeField.getText();
 		Label phoneLabel = new Label("Phone:");
 		phoneField = new TextField();
+		String phone = phoneField.getText();
 		Label newUsernameLabel = new Label("User Name:");
 		newUsernameField = new TextField();
+		String userName = newUsernameField.getText();
 		Label newPasswordLabel = new Label("Password:");
 		newPasswordField = new TextField();
+		String password = newPasswordField.getText();
 		Label rePasswordLabel = new Label("Re-enter Password:");
 		rePasswordField = new TextField();
+		String rePassword = rePasswordField.getText();

```

Comment:
For this as well, just a loop to make sure the password has been entered correctly.